packages/evm/src/accessWitness.ts

```diff
@@ -43,17 +44,15 @@ type RawAccessedState = {
   chunkKey: PrefixedHexString
 }
 
-export enum AccessedStateType {
-  BasicData = 'basicData',
-  CodeHash = 'codeHash',
-  Code = 'code',
-  Storage = 'storage',
-}
-
 type AccessedState =
-  | { type: Exclude<AccessedStateType, AccessedStateType.Code | AccessedStateType.Storage> }
-  | { type: AccessedStateType.Code; codeOffset: number }
-  | { type: AccessedStateType.Storage; slot: bigint }
+  | {
+      type: Exclude<
+        VerkleAccessedStateType,
+        VerkleAccessedStateType.Code | VerkleAccessedStateType.Storage
+      >
+    }
+  | { type: VerkleAccessedStateType.Code; codeOffset: number }
+  | { type: VerkleAccessedStateType.Storage; slot: bigint }
 export type AccessedStateWithAddress = AccessedState & {
```

Comment:
Shouldn't these types move to `common` or else `util`?  We need them in the verkle state managers and can't import them from `evm` since that would create a circular dep.

Reply:
Yeah agreed, I really haven't done much work yet (just moved the few obvious parts yesterday before going to bed), probably best to wait a bit before reviewing